src/references/index.ts

```diff
@@ -217,6 +218,7 @@ export const NATIVE_ASSETS_MAP_PER_CHAIN: Record<ChainId, AddressOrEth> = {
   [ChainId.sepolia]: ETH_ADDRESS,
   [ChainId.zora]: ETH_ADDRESS,
   [ChainId.zoraSepolia]: ETH_ADDRESS,
+  [ChainId.degen]: AddressZero,
```

Comment:
for consistency can we use `ETH_ADDRESS` here. Don't think it's functionally any different but we're using it for other chains so would like to keep it consistent.

Reply:
just noticed this `NATIVE_ASSETS_MAP_PER_CHAIN` is not used anywhere, what was it for? 

in `NATIVE_ASSETS_PER_CHAIN` it gonna be `AddressZero` to work right

Reply:
I think it was moved to the swaps sdk iirc. Maybe NATIVE_ASSETS_PER_CHAIN was too?